e2e/hits_test/test_cugraph_10_nodes/input.cyp

```diff
@@ -0,0 +1,17 @@
+CREATE INDEX ON :Node(id);
```

Comment:
Should we add CREATE INDEX on each test here?

Reply:
No real speedup with index on test with 10 nodes :smile: